src/base58.cpp

```diff
@@ -44,8 +62,11 @@ bool DecodeBase58(const char* psz, std::vector<unsigned char>& vch)
         zeroes++;
         psz++;
     }
+    // Decoding a megabyte of Base58 takes minutes, seems like a reasonable upper limit.
+    size_t len = strlen(psz);
+    if (len > 1000000) return false;
```

Comment:
Let's make `1000000` a constant, like `BASE58_LIMIT`.

Reply:
I've rewritten it, this constant is no longer necessary.